website-frontend/tests/layout.spec.ts

```diff
@@ -1,15 +1,17 @@
 import { test, expect } from '@playwright/test';
 
-const EXPECTED_SITE_TITLE = 'UP Diliman - Department of Computer Science';
-
 test.describe('2-4: Navigation Bar Links to Home', () => {
-	test('Check for home button in navbar', async ({ page }) => {
+	test('Check for home button in navbar', async ({ page, request }) => {
 		await page.goto('/');
 		const nav = await page.getByRole('navigation');
 		const home = await nav.getByRole('link', { name: 'Home' });
 		await home.click();
 		await page.waitForTimeout(5000);
-		expect(await page.title()).toBe(EXPECTED_SITE_TITLE);
+		const res = await request.get(`${process.env.PUBLIC_APIURL}/items/global`);
+		const global_title = await res.json().then((obj) => {
```

Comment:
Could be nice to use `.catch` as well to handle errors upon parsing using `json()`, but I think it's good for merging!

Let me know if you plan to enact that feedback or stick with the current implementation, so that we could merge this right away.

Reply:
Hi! For now I think I'm pretty good without the error catching, we can get this merged in now. Thanks!